cursorless-talon/src/spoken_forms.json

```diff
@@ -233,32 +233,6 @@
       "-from": "experimental.setInstanceReference"
     }
   },
-  "experimental/wrapper_snippets.csv": {
```

Comment:
I wonder if we want to be more explicit. Just allowing us to leave out things feels like it's making it easier to make mistakes. Maybe we use a string `"DEPRECATED"`?

Reply:
We still need to specify the file name otherwise we don't know which csv file to read if it exists. I could of course add an extra argument to init_csv_and_watch_changes instead of passing none as the default values?

Reply:
I went ahead with a boolean solution